src/civis/resources/_resources.py

```diff
@@ -334,7 +335,7 @@ def raise_for_unexpected_kwargs(
 
 
 def bracketed(x):
-    return re.search("^{.*}$", x)
+    return _BRACKETED_REGEX.search(x)
```

Comment:
On the theme of speeding things up and as I saw regex from `camel_to_snake` above, I've noticed we aren't using compiled regex expressions where possible, but we should. Many of these little functions in this module are heavily used in handling an API spec, so we should optimize them:

```python
In [1]: import re

In [2]: %timeit re.search(r"^{.*}$", "foobar")
521 ns ± 4.13 ns per loop (mean ± std. dev. of 7 runs, 1,000,000 loops each)

In [3]: _BRACKETED_REGEX = re.compile(r"^{.*}$")

In [4]: %timeit _BRACKETED_REGEX.search("foobar")
159 ns ± 1.84 ns per loop (mean ± std. dev. of 7 runs, 10,000,000 loops each)
```